bridge/client/app/_components/ktb-approval-item/ktb-approval-item.component.ts

```diff
@@ -46,4 +48,8 @@ export class KtbApprovalItemComponent {
     this.approvalResult = result;
   }
 
+  public getDeploymentEvaluation(project: Project): Trace | undefined {
+    return this.isSequence ? project.getDeploymentEvaluationOfSequence(this.event) : project.getDeploymentEvaluation(this.event);
+  }
+
```

Comment:
What is the difference between `getDeploymentEvaluationOfSequence` and `getDeploymentEvaluation` and can't we handle this inside the `getDeploymentEvaluation` method in the `project`-model? If necessary add another parameter, but I think it is not necessary and in long-term we should use the `sequence` entity instead of the `root` entity.